src/steps/os/unix.rs

```diff
@@ -84,3 +84,51 @@ pub fn run_pearl(run_type: RunType) -> Result<(), Error> {
 
     run_type.execute(&pearl).arg("update").check_run()
 }
+
+pub fn run_sdkman(base_dirs: &BaseDirs, cleanup: bool, run_type: RunType) -> Result<(), Error> {
+    let bash_path = require("bash").map(|p| format!("{}", &p.display()))?;
```

Comment:
I think you can remove the `.map` and have it stay as `PathBuf`

Reply:
Yes you are right. I'll remove it and make a new pull request.